tests/components/do_verify_email.test.jsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+
+import Adapter from 'enzyme-adapter-react-16';
+
+import {verifyUserEmail as VerifyUserMail} from 'mattermost-redux/actions/users';
```

Comment:
I suggests not to import this and would recommend only doing the mockup of `props.verifyUserEmail`.

Would prefer to have tests as well for:
- on verifyUserEmail success (with state and browserHistory.push assertions) similar to this - https://github.com/mattermost/mattermost-webapp/blob/master/tests/components/integrations/edit_oauth_app.test.jsx#L139
- on verifyUserEmail error (with state assertions) similar to this - https://github.com/mattermost/mattermost-webapp/blob/master/tests/components/integrations/edit_oauth_app.test.jsx#L165